enterprise/metrics/src/main/java/org/neo4j/metrics/docs/MetricsAsciiDocGenerator.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.metrics.docs;
+
+import java.lang.reflect.Field;
+
+import org.neo4j.kernel.impl.annotations.Documented;
+
+public class MetricsAsciiDocGenerator
```

Comment:
Good question. I don't know what is the best. This is in main becaushttps://github.com/neo4j/neo4j/blob/3.0/community/kernel/src/main/java/org/neo4j/kernel/configuration/ConfigAsciiDocGenerator.java is in main as well.